lib/gitlib.sh

```diff
@@ -24,6 +24,7 @@ GHCURL="curl -s --fail --retry 10 -u ${GITHUB_TOKEN:-$FLAGS_github_token}:x-oaut
 JCURL="curl -g -s --fail --retry 10"
 K8S_GITHUB_API='https://api.github.com/repos/kubernetes/kubernetes'
 K8S_GITHUB_URL='https://github.com/kubernetes/kubernetes'
+K8S_GITHUB_SSH='git@github.com:kubernetes/kubernetes.git'
```

Comment:
Will this create kubernetes.git directories when the target is unspecified?  Maybe leave this without the .git for compatibility.